src/lib/ViewerState.ts

```diff
@@ -24,8 +25,11 @@ export class ViewerState {
     showTracks = true;
     showTrackHighlights = true;
     // Default position and target for the camera
-    cameraPosition = [0, 0, -4]; // was  [500, 500, -1250];
-    cameraTarget = [0, 0, 0]; // was  [500, 500, 500];
+    cameraPosition = [0, 0, -4];
+    cameraTarget = [0, 0, 0];
+    pointSize: number = initialPointSize;
+    trackWidth: number = 0.5;
```

Comment:
Is this ever used?

Reply:
They are used in `PointCanvas.ts/toState`, to ensure these settings are taken into the hash of the configuration state, when copied as URL.

Reply:
I see that `trackWidthFactor` is returned by `toState` and used for rendering, but don't see the same for `trackWidth`.